benchmarks/dc_eis_prefootprint/dceisPrefootprint.json

```diff
@@ -0,0 +1,13 @@
+{
+ "benchid": 12,
```

Comment:
This PR is missing the part to add the benchids for the 2 new benchmarks.  12 is also now taken so this id will need to be updated.